controllers/doguInstallManager.go

```diff
@@ -285,6 +293,37 @@ func (m *doguInstallManager) createConfigs(ctx context.Context, doguName string,
 	return cleanUp, nil
 }
 
+// SetCurrentDataVolumeSize set the default DataVolumeSize within the status of the dogu
+func SetDefaultDataVolumeSize(ctx context.Context, client client.Client, doguResource *doguv2.Dogu) error {
```

Comment:
The function would benefit if the name says that the VolumeSize will be set in the status.

Like: SetSpecVolumeSizeInStatus or SetDefaultDataVolumeSizeInStatus

Keep in mind. It is not always the default size